sys/src/cmd/preen/.gitignore

```diff
@@ -0,0 +1 @@
+/preen
```

Comment:
this isn't right

Reply:
why?
it's saying the preen binary in this directory should be ignored.

Reply:
If the pattern contains a slash, which is not in the trailing position (it's not the last character), then it is matched only with pathnames relative to the .gitignore file location.